File: cs50/sentimental-cash/cash.py
from cs50 import get_float

cent = int(get_float("Change: ")*100)
while cent<0:
    cent = int(get_float("Change: ")*100)
owned=0
while cent!=0:
    if cent>=25:
        owned+=cent//25
        cent=cent%25
    elif cent>=10:
        owned+=cent//10
        cent=cent%10
    elif cent>=5:
        owned+=cent//5
        cent=cent%5
    else:
        owned+=cent
        cent=0
print(owned)

File: cs50/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int money, change = 0;
    do
    {
        money = get_int("Change owed: ");
    }
    while (money <= 0);

    while (money != 0)
    {
        if (money >= 25)
        {
            money -= 25;
            change++;
        }
        else if (money >= 10)
        {
            money -= 10;
            change++;
        }
        else if (money >= 5)
        {
            money -= 5;
            change++;
        }
        else if (money >= 1)
        {
            money -= 1;
            change++;
        }
    }
    printf("%d\n", change);
    return 0;
}
